iib/workers/tasks/build.py

```diff
@@ -386,7 +386,9 @@ def _opm_index_add(
 
 
 @retry(attempts=2, wait_on=IIBError, logger=log)
-def _opm_index_rm(base_dir, operators, binary_image, from_index, overwrite_from_index_token=None):
+def _opm_index_rm(
```

Comment:
Could you please update a call of this function (on line 870). Otherwise a wrong value is passed to the `ocp_version` parameter.
Thank you.